pac_windows.go

```diff
@@ -0,0 +1,91 @@
+package ieproxy
+
+import (
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+type WINHTTP_AUTOPROXY_OPTIONS struct {
```

Comment:
please rename this `tWINHTTP_AUTOPROXY_OPTIONS`, or move this to internal package with followings.